javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,39 @@
+//A future calculator
+const yearOfBirth = 1984;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}`)
+
+//A dog age calculator 
+const dogYearOfBirth = 2020;
+let dogYearFuture = 2045;
+let dogYear = (dogYearFuture - dogYearOfBirth)/2*10;
+let shouldShowResultInDogYears = false;
+
+if(shouldShowResultInDogYears) {
+console.log(`Your dog will be ${dogYear} human years old in ${dogYearFuture}`);
+} else {
+    console.log(`Your dog will be ${dogYearFuture - dogYearOfBirth} dogs years old in ${dogYearFuture}`);  
+}
+
+//A house price estimation 
+let houseCosts = 2500000;
```

Comment:
Nit:
since parameters of the house forms one logical entity ("house") it would be good idea to use one object instead of multiple variables for each piece of information. 
e.g. 
```
const house  = {
   wideMeters: 8, 
   deepMeters: 10,
   highMeters: 10,
   gardenSizeInMeter2 : 100
  price: 2500000,
}
const volumeInMeters  = house.wideMeters * house.deepMeters * highMeters
const housePrice  = volumeInMeter * 2.5 .... 
if (housePrice < house.price) {
 ....
} else {
...
}

```